From 619bba935e368193a6e86a779c385e18f78c1d26 Mon Sep 17 00:00:00 2001
Message-Id: <619bba935e368193a6e86a779c385e18f78c1d26.1368111913.git.minovotn@redhat.com>
In-Reply-To: <405603258af5154387bea676be1f904b6713f6ae.1368111913.git.minovotn@redhat.com>
References: <405603258af5154387bea676be1f904b6713f6ae.1368111913.git.minovotn@redhat.com>
From: Amit Shah <amit.shah@redhat.com>
Date: Wed, 24 Apr 2013 08:17:55 +0200
Subject: [PATCH 21/65] char: rename qemu_chr_set_echo() ->
 qemu_chr_fe_set_echo()

RH-Author: Amit Shah <amit.shah@redhat.com>
Message-id: <78e72e7b893095f776da8165bbe8059e71684816.1366724981.git.amit.shah@redhat.com>
Patchwork-id: 50799
O-Subject: [RHEL6.5 qemu-kvm PATCH 21/65] char: rename qemu_chr_set_echo() -> qemu_chr_fe_set_echo()
Bugzilla: 909059
RH-Acked-by: Hans de Goede <hdegoede@redhat.com>
RH-Acked-by: Gerd Hoffmann <kraxel@redhat.com>
RH-Acked-by: Paolo Bonzini <pbonzini@redhat.com>

From: Anthony Liguori <aliguori@us.ibm.com>

Signed-off-by: Anthony Liguori <aliguori@us.ibm.com>
(cherry picked from commit 15f31519b423747285bdc924ec4138c554076a73)

Signed-off-by: Amit Shah <amit.shah@redhat.com>

Conflicts:
	qemu-char.c
	qemu-char.h

Signed-off-by: Amit Shah <amit.shah@redhat.com>
---
 monitor.c   |  2 +-
 qemu-char.c | 16 ++++++++--------
 qemu-char.h |  2 +-
 3 files changed, 10 insertions(+), 10 deletions(-)

Signed-off-by: Michal Novotny <minovotn@redhat.com>
---
 monitor.c   |  2 +-
 qemu-char.c | 16 ++++++++--------
 qemu-char.h |  2 +-
 3 files changed, 10 insertions(+), 10 deletions(-)

diff --git a/monitor.c b/monitor.c
index 6625a46..7cd5aeb 100644
--- a/monitor.c
+++ b/monitor.c
@@ -5122,7 +5122,7 @@ void monitor_init(CharDriverState *chr, int flags)
         /* Control mode requires special handlers */
         qemu_chr_add_handlers(chr, monitor_can_read, monitor_control_read,
                               monitor_control_event, mon);
-        qemu_chr_set_echo(chr, true);
+        qemu_chr_fe_set_echo(chr, true);
     } else {
         qemu_chr_add_handlers(chr, monitor_can_read, monitor_read,
                               monitor_event, mon);
diff --git a/qemu-char.c b/qemu-char.c
index cf0d4ed..973b394 100644
--- a/qemu-char.c
+++ b/qemu-char.c
@@ -786,7 +786,7 @@ static CharDriverState *qemu_chr_open_stdio(QemuOpts *opts)
     stdio_nb_clients++;
     stdio_allow_signal = qemu_opt_get_bool(opts, "signal",
                                            display_type != DT_NOGRAPHIC);
-    qemu_chr_set_echo(chr, false);
+    qemu_chr_fe_set_echo(chr, false);
 
     return chr;
 }
@@ -2575,6 +2575,13 @@ CharDriverState *qemu_chr_open(const char *label, const char *filename, void (*i
     return chr;
 }
 
+void qemu_chr_fe_set_echo(struct CharDriverState *chr, bool echo)
+{
+    if (chr->chr_set_echo) {
+        chr->chr_set_echo(chr, echo);
+    }
+}
+
 void qemu_chr_fe_open(struct CharDriverState *chr)
 {
     if (chr->chr_guest_open) {
@@ -2589,13 +2596,6 @@ void qemu_chr_fe_close(struct CharDriverState *chr)
     }
 }
 
-void qemu_chr_set_echo(struct CharDriverState *chr, bool echo)
-{
-    if (chr->chr_set_echo) {
-        chr->chr_set_echo(chr, echo);
-    }
-}
-
 void qemu_chr_close(CharDriverState *chr)
 {
     QTAILQ_REMOVE(&chardevs, chr, next);
diff --git a/qemu-char.h b/qemu-char.h
index e6f7eab..c4413ec 100644
--- a/qemu-char.h
+++ b/qemu-char.h
@@ -80,9 +80,9 @@ QemuOpts *qemu_chr_parse_compat(const char *label, const char *filename);
 CharDriverState *qemu_chr_open_opts(QemuOpts *opts,
                                     void (*init)(struct CharDriverState *s));
 CharDriverState *qemu_chr_open(const char *label, const char *filename, void (*init)(struct CharDriverState *s));
+void qemu_chr_fe_set_echo(struct CharDriverState *chr, bool echo);
 void qemu_chr_fe_open(struct CharDriverState *chr);
 void qemu_chr_fe_close(struct CharDriverState *chr);
-void qemu_chr_set_echo(struct CharDriverState *chr, bool echo);
 void qemu_chr_close(CharDriverState *chr);
 void qemu_chr_fe_printf(CharDriverState *s, const char *fmt, ...);
 int qemu_chr_fe_write(CharDriverState *s, const uint8_t *buf, int len);
-- 
1.7.11.7

